lunamark/reader/markdown.lua

```diff
@@ -1050,10 +1170,28 @@ function M.new(writer, options)
                        = (parsers.TildeFencedCodeBlock
                          + parsers.BacktickFencedCodeBlock)
                        / function(infostring, code)
+                           if options.pandoc_extensions then
+                             local raw, attr = lpeg.match(parsers.rawattributes, infostring)
```

Comment:
Can we do this without nested parsers?

Reply:
I haven't found a clear way yet (even trickier now with the split options)